packages/dialogs/src/RPA/Dialogs/library.py

```diff
@@ -16,7 +16,12 @@
 
 @library(scope="GLOBAL", doc_format="REST", auto_keywords=False)
 class Dialogs:
-    """The `Dialogs` library provides a way to display information to a user
+    """**NOTICE:** The `Dialogs` library is now marked for deprecation.
+    The library will be replaced by `Assistant`. In order to get more
+    information and check the changes and the migration guide, please
+    check the Release Notes: https://rpaframework.org/releasenotes.html#id4
```

Comment:
```suggestion
    check the Release Notes: https://updates.robocorp.com/release/Gfko5-rpaframework-2200
```
or
```suggestion
    check the release updates: https://updates.robocorp.com/
```

Beware that the .org release notes aren't stable and they may change links when you add a new one or change the file.

Reply:
Oh, interesting, Kari requested this URL, but I guess he also didn't knew it can change, thank you!